mkdocs/pages/en/devbook/accounts/account-metadata.md

```diff
@@ -0,0 +1,202 @@
+  [by using a wallet](../../userbook/wallet/create-account.md).
+- Obtain <XYM:> to pay for the transaction fee.
+  See [Getting Testnet Funds from the Faucet](./testnet-faucet.md).
+
+Additionally, review the [Transfer transaction](../transactions/transfer.md) tutorial to understand how
+transactions are announced and confirmed, and the
+[Complete Aggregate transaction](../transactions/complete-aggregate.md) tutorial to understand how aggregate
+transactions work.
+
+## Full Code
+
+{% import 'tutorial.jinja2' as tutorial with context %}
+
+{{ tutorial.code_full('devbook/transactions/account-metadata', ['py', 'js']) }}
+
+## Code Explanation
+
+This tutorial demonstrates adding new metadata to an account and then updating that metadata.
+
+### Setting Up the Account
+
+{{ tutorial.code_snippet(['py:53:61', 'js:51:59']) }}
+
+The snippet reads the signer's private key from the `SIGNER_PRIVATE_KEY` environment variable, which defaults to a
+test key if not set.
+The signer's address is derived from the public key.
+
+In this tutorial, the signer adds metadata to their own account.
+Adding metadata to a different account requires the target to cosign the transaction.
+
+### Fetching Network Time and Fees
+
+{{ tutorial.code_snippet(['py:64:82', 'js:62:80']) }}
+
+Network time and recommended fees are fetched from <get:/node/time> and <get:/network/fees/transaction> respectively, following the process described in the [Transfer Transaction](./transfer.md) tutorial.
+
+### Defining the Metadata
+
+{{ tutorial.code_snippet(['py:84:87', 'js:82:85']) }}
+
+Each metadata entry is identified by a **scoped metadata key**, a 64-bit value chosen by the metadata creator.
+The combination of the signer's address, the target account's address, and the scoped metadata key uniquely
+identifies the metadata entry.
+
+!!! note "Multiple entries with the same key"
+
+    Because the signer's address is part of the unique identifier, multiple accounts can add metadata with the same
+    scoped key to a target (e.g., Account A and Account B can add a different `username` to Account C).
+    Each entry remains separate and can only be updated by its signer.
+
+The SDK provides a <dy:metadataGenerateKey> helper function that generates a key from a
+human-readable string using SHA3-256 hashing.
+This approach makes keys more meaningful and reduces the chance of collisions.
+
+In this example, the key is derived from the string `username`.
+To ensure the metadata key is unique across multiple runs of the tutorial, a timestamp is appended to the key string.
+In practice, you would use a stable key that identifies the specific metadata entry you want to create or update.
+
+The metadata value can be any byte sequence.
+In this example, the username `alice` is UTF-8 encoded.
+
+### Creating the Embedded Metadata Transaction
+
+{{ tutorial.code_snippet(['py:89:101', 'js:87:100']) }}
+
+Symbol requires metadata transactions to be inside an <aggregate transaction:>.
```

Comment:
It would be good to explain why.
I understand this is useful when the signer and the target accounts are different, but it looks cumbersome when they are not.
The Symbol Whitepaper does not mention metadata anywhere, so you'll have to ask in the chat.